packages/db/.env.example

```
@@ -0,0 +1,15 @@
+########################
+# Local Postgres (Docker Compose)
+########################
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=yourpasswordlocalhost
+POSTGRES_DB=postgres
+
+########################
+# Prisma (CLI, db:studio, app runtime)
+########################
+DATABASE_URL=postgresql://postgres:yourpasswordlocalhost@localhost:5432/postgres
+DIRECT_URL=postgresql://postgres:yourpasswordlocalhost@localhost:5432/postgres
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)